index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Ja'fer's Portfolio</title>
+</head>
+<body>
+  <nav class="header">
+    
+    <p class="welcome"> <strong>Welcome</strong></p>
+    <div class="vector_container">
+      <a href="">
+      <svg width="40" height="3">
+        <rect class="vector" width="40" height="3" />
+      </svg>
+    </a>
+    <a href="">
+      <svg width="40" height="3">
+        <rect class="vector" width="40" height="3"  />
+      </svg>
+    </a>
+    <a href="">
+      <svg width="40" height="3">
+        <rect class="vector" width="40" height="3"  />
+      </svg>
+    </a>
+    </div>
```

Comment:
- [ ] Please add your styling in the external CSS folder, don't use inline style.  This will make your codes more clean and organized. having styling and Html in separate files.